padertorch/contrib/examples/speaker_classification/supervised/evaluate.py

```diff
@@ -0,0 +1,166 @@
+"""
+Example calls:
+
+On single CPU (slow):
+python -m padertorch.contrib.examples.speaker_classification.supervised.evaluate with model_path=<path/to/trained/model>
+
+On GPU:
+python -m padertorch.contrib.examples.speaker_classification.supervised.evaluate with model_path=<path/to/trained/model> device=0 batch_size=16
+
+On multiple CPUs:
+mpiexec -np 8 python -m padertorch.contrib.examples.speaker_classification.supervised.evaluate with model_path=<path/to/trained/model>
+"""
+import os
+from pathlib import Path
+import warnings
+
+from sacred import Experiment, commands
+import numpy as np
+import torch
+
+from dlp_mpi import COMM, IS_MASTER, MASTER, split_managed
+
+from paderbox.io.new_subdir import get_new_subdir
+from paderbox.io import load_json, dump_json
+
+import padertorch as pt
+
+from .data import get_datasets
+
+ex = Experiment('speaker_clf')
+
+
+@ex.config
+def defaults():
+    model_path = None
+    assert model_path is not None, (
+        'model_path cannot be None.\n'
+        'Start the evaluation with "python -m padertorch.contrib.examples.'
+        'speaker_classification.evaluate with model_path=<path/to/trained/model>"'
+    )
+    load_ckpt = 'ckpt_best_loss.pth'
+    batch_size = 1
+    device = 'cpu'
```

Comment:
You could change this to `0 if torch.cuda.is_available() else 'cpu'`. Then it automatically detects the gpu. (i.e. it reads CUDA_VISIBLE_DEVICES)

Reply:
https://github.com/fgnt/padertorch/pull/57/commits/22310de5eafca150b8fcd944034768e0abcf8d98